saleor/attribute/utils.py

```diff
@@ -114,12 +114,21 @@ def _associate_attribute_to_instance(
             instance, instance_attrs_ids, AssignedVariantAttribute, instance_field_name
         )
 
+    # workaround for checking uniqness using database check
```

Comment:
Could you include information about such tasks or related work in the PR description?